src/policy_metadata.rs

```diff
@@ -120,6 +136,27 @@ fn validate_resources(data: &[String]) -> Result<(), ValidationError> {
     Ok(())
 }
 
+impl TryFrom<&NamedRuleWithOperations> for Rule {
+    type Error = &'static str;
+
+    fn try_from(rule: &NamedRuleWithOperations) -> Result<Self, Self::Error> {
+        let operations: Result<Vec<Operation>, Self::Error> = match &rule.operations {
+            Some(operations) => operations
+                .iter()
+                .map(|op| Operation::try_from(op.as_str()))
+                .collect(),
+            None => Ok(Vec::new()),
+        };
```

Comment:
nit, you could use the question mark here instead

Reply:
Done